pkgs/development/compilers/dotnet/vmr.nix

```diff
@@ -156,7 +156,8 @@ stdenv.mkDerivation rec {
   postPatch =
     ''
       # set the sdk version in global.json to match the bootstrap sdk
-      jq '(.tools.dotnet=$dotnet)' global.json --arg dotnet "$(${bootstrapSdk}/bin/dotnet --version)" > global.json~
+      sdk_version=$(HOME= ${bootstrapSdk}/bin/dotnet --version)
```

Comment:
10.0-preview.2 returns an error code when $HOME points at a non-existent path.

Reply:
I ended up using a temporary directory for this, because `HOME=` just makes it fall back to using `/etc/passwd`, which doesn't work in the darwin sandbox.